doc/development/plugins.rst

```diff
@@ -43,54 +58,67 @@ This is as simple as importing the abstract base class :class:`~.Device` from Pe
-* :attr:`~.Device.name`: a string containing the official name of the device
+* :attr:`.Device.name`: a string containing the official name of the device
 
-* :attr:`~.Device.short_name`: the string used to identify and load the device by users of PennyLane
+* :attr:`.Device.short_name`: the string used to identify and load the device by users of PennyLane
 
-* :attr:`~.Device.pennylane_requires`: the PennyLane version this device supports. Note that this class attribute supports pip *requirements.txt* style version ranges, for example:
+* :attr:`.Device.pennylane_requires`: the PennyLane version this device supports.
+  Note that this class attribute supports pip *requirements.txt* style version ranges,
+  for example:
 
   - ``pennylane_requires = "2"`` to support PennyLane version 2.x.x
   - ``pennylane_requires = ">=0.1.5,<0.6"`` to support a range of PennyLane versions
 
-* :attr:`~.Device.version`: the version number of the device
+* :attr:`.Device.version`: the version number of the device
 
-* :attr:`~.Device.author`: the author of the device
+* :attr:`.Device.author`: the author of the device
 
 Defining all these attributes is mandatory.
 
 .. note::
 
-    For examples of subclasses of :class:`Device`, see :class:`~.DefaultQubit`,
-    :class:`~.DefaultGaussian`, or the `StrawberryFields <https://pennylane-sf.readthedocs.io/>`_
-    and `ProjectQ <https://pennylane-pq.readthedocs.io/>`_ plugins.
+    For examples of subclasses of :class:`~.Device`, see :class:`~.DefaultQubit`,
+    :class:`~.DefaultGaussian`, or the `Qiskit <https://pennylane-qiskit.readthedocs.io/>`_
+    plugin.
 
 
 Supporting operators and observables
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+------------------------------------
 
-You must further tell PennyLane about the operations and observables that your device supports as well as potential further capabilities, by providing the following class attributes/properties:
+You must further tell PennyLane about the operations and observables that your device supports
+as well as potential further capabilities, by providing the following class attributes/properties:
 
-* :attr:`~.Device.operations`: a set of the supported PennyLane operations as strings, e.g.,
+* :attr:`.Device.operations`: a set of the supported PennyLane operations as strings, e.g.,
 
   .. code-block:: python
 
     operations = {"CNOT", "PauliX"}
 
-  This is used to decide whether an operation is supported by your device in the default implementation of the public method :meth:`~.Device.supports_operation`.
+  This is used to decide whether an operation is supported by your device in the
+  default implementation of the public method :meth:`.Device.supports_operation`.
 
-* :attr:`~.Device.observables`: set of the supported PennyLane observables as strings, e.g.,
+* :attr:`.Device.observables`: set of the supported PennyLane observables as strings, e.g.,
 
   .. code-block:: python
 
     observables = {"QuadOperator", "NumberOperator", "X", "P"}
```

Comment:
Maybe one should not mix qubit operations and cv observables in the explanation? That could confuse people.